### PR TITLE
validations: refactor files names and handlers

### PR DESCRIPTION
**Pull request by MarcoWorms**
The old validations file structure and names was over-verbose and had
some unecessary code. This refactor tries to clean up most of the logic
that unites the single validations in larger ones while also renaming
most of the `card` validation that was unecessarily verbose as well.

### CODE REVIEW

**Comment by felquis** [comment]
These are a lot of changes!!!

**Comment by MarcoWorms** [comment]
It's just renaming so it seems a lot

**Comment by MarcoWorms** [comment]
@felquis @derekstavis @LeoNero 

can any of you take a look at this PR? It seems like a lot but it's mostly renaming.

File: lib/validations/validate/card/index.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+const validateAll = card => applySpec({
+  brand: getBrand,
+  card_holder_name: validHolder,
+  card_number: validCardNumber,
+  card_expiration_date: validDate,
+  card_cvv: validCvv(getBrand(card)),
+})(card)
+
+const validateExistentProps = card =>
+  omit(...ap([missingProps, validateAll], [card]))
```
Comment (mccraveiro) [inline lib/validations/validate/card/index.js]: @MarcoWorms use `apply` instead of `...`

File: lib/validations/index.spec.js [hunk 2]
```diff
@@ -1,8 +1,9 @@
+
```
Comment (mccraveiro) [inline lib/validations/index.spec.js]: Extra line

File: lib/validations/validate/card/index.js (hunk 1 above)
Comment (MarcoWorms) [inline lib/validations/validate/card/index.js]: OH YOU

File: lib/validations/validate/card/brand.js [hunk 3]
```diff
@@ -31,18 +32,18 @@ const cardNumberMatchBins = (cardNumber, brandBins) =>
 const makeCardMatcher = cardNumber => brandBins =>
   cardNumberMatchBins(cardNumber, brandBins)
 
-const getBrand = pipe(
+const brand = pipe(
   makeCardMatcher,
   mapBins,
   toPairs,
-  find(pair => last(pair) === true),
```
Comment (MarcoWorms) [inline lib/validations/validate/card/brand.js]: @piiih mudei uma linha sua aqui pra ficar "ramda way" :D

**Pull request merged**